### PR TITLE
Some more fixes to make it work on the Raspi again

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by ChrisBr** [comment]
Note that the rake file is quick and dirty but wanted to push it somewhere for the next time ;)

File: lib/tasks/picture_set.rake [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+        Syscall.execute("cp #{File.join(path, date, "*.gif")} #{args[:output]}")
+      rescue => e
+        puts e
+    end
+  end
+  
+  def combine_images(date, path)
+  dir = File.join(path, date)
+  begin
+  Syscall.execute("time montage -geometry '25%x25%+25+25<' -background '#{OPTS.background_color}' " \
```
Comment (digitaltom) [inline lib/tasks/picture_set.rake]: Better only do it when `#{date}#{COMBINED_SUFFIX}` does not exist yet.

**Comment by ChrisBr** [comment]
@digitaltom added the skip for combined image tasks

**Comment by ChrisBr** [comment]
Rubcop is acting weird :thinking: No idea how this one line introduced 

```
Offenses:
lib/tasks/picture_set.rake:3:1: W: Lint/UnneededCopDisableDirective: Unnecessary disabling of Metrics/BlockLength.
# rubocop:disable Metrics/BlockLength
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
lib/tasks/picture_set.rake:57:5: E: Lint/Syntax: unexpected token kBEGIN
(Using Ruby 2.1 parser; configure using TargetRubyVersion parameter, under AllCops)
    begin
    ^^^^^
```

**Comment by ChrisBr** [comment]
> Rubcop is acting weird thinking No idea how this one line introduced

Ok, there was a ``)`` missing but the error message was really strange :/

**Pull request merged**